src/format/Format.cc

```diff
@@ -1431,6 +1432,13 @@ Format::Format::assemble(MemBuf &mb, const AccessLogEntry::Pointer &al, int logS
             if (!al->lastAclData.isEmpty())
                 out = al->lastAclData.c_str();
             break;
+
+        case LFT_MASTER_XACTION:
+            if (al->request) {
+                sb = ToSBuf(al->request->masterXaction->id);
```

Comment:
Should this be formatted as a number instead of a string? See doint and similar variables. We may need to add a `douint64` variable, but that is a secondary question/problem. The primary decision here is whether the ID is a number or a string.

Reply:
SBuf and ToSBuf approach is better IMO: this will work regardless of what value type masterXaction->id bears.

Reply:
Before we consider implementation pros and cons, we should decide on the _interface_: Would admins prefer the transaction ID to be a number or a string?

Reply:
Done at ea81d67.